cassandra/roles/install-cassandra/templates/cassandra-app.yaml.j2

```diff
@@ -0,0 +1,91 @@
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: cassandra
+  namespace: {{ cassandra_namespace }}
+  labels:
+    app: cassandra
+spec:
+  serviceName: cassandra
+  replicas: 3
+  selector:
+    matchLabels:
+      app: cassandra
+  template:
+    metadata:
+      labels:
+        app: cassandra
+    spec:
+      terminationGracePeriodSeconds: 1800
+      containers:
+        - name: cassandra
+          image: cassandra:3
```

Comment:
We should also try testing with:  https://hub.docker.com/r/centos/cassandra-3-centos7/. This may be how we get around the privileged SCC bits.